subsys/logging/log_core.c

```diff
@@ -182,24 +182,11 @@ static void detect_missed_strdup(struct log_msg *msg)
 #undef ERR_MSG
 }
 
-static inline void msg_finalize(struct log_msg *msg,
-				struct log_msg_ids src_level)
+void z_log_msg_post_finalize(void)
```

Comment:
`static void` Shouldn't it has limited visibility scope ?

Reply:
fixed